hack/patch.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+
+kubectl=./cluster-up/kubectl.sh
+
+function wait-for-daemonset() {
+    retries=10
+    while [[ $retries -ge 0 ]]; do
+        sleep 3
+        ready=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.numberReady}")
+        required=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.desiredNumberScheduled}")
+        if [[ $ready -eq $required ]]; then
+            #echo "Succeeded"
+            break
+        fi
+        ((retries--))
+    done
+}
+
+BAZEL=bazelisk PUSH_TARGETS="virt-controller virt-handler virt-launcher" ./hack/bazel-push-images.sh
+
+registry_port=$(./cluster-up/cli.sh ports registry)
+registry_url=localhost:$registry_port
+
+handler_sha256=$(skopeo inspect docker://$registry_url/kubevirt/virt-handler:latest --tls-verify=false | jq ".Digest" -r)
+$kubectl set env deployment -n kubevirt virt-operator VIRT_HANDLER_SHASUM=$handler_sha256
+
+launcher_sha256=$(skopeo inspect docker://$registry_url/kubevirt/virt-launcher:latest --tls-verify=false | jq ".Digest" -r)
+$kubectl set env deployment -n kubevirt virt-operator VIRT_LAUNCHER_SHASUM=$launcher_sha256
+
+controller_sha256=$(skopeo inspect docker://$registry_url/kubevirt/virt-controller:latest --tls-verify=false | jq ".Digest" -r)
+$kubectl set env deployment -n kubevirt virt-operator VIRT_CONTROLLER_SHASUM=$controller_sha256
+
+# Force virt-handler restart to ensure is taking virt-launcher changes
+$kubectl delete ds -n kubevirt virt-handler
```

Comment:
If the change is automatically taken for deployments but not for virt-handler, that sounds like a bug in the operator. @ashleyschuett FYI.

Reply:
I found that when launcher is updated virt-handler need to be restarted too, maybe it's a bug.

Reply:
This sounds strange. Do you have more insights on that? Maybe it is related to already started VMIs when you do incompatible changes?